lib/generator.js

```diff
@@ -195,10 +205,21 @@ const generateMd = (inventory, options = {}) => {
   for (const [name, methods] of inventory) {
     if (!options.removeInterface) buf.push(`### Interface: ${name}\n`);
     for (const [method, signature] of methods) {
-      const args = signature.parameters
```

Comment:
I think it's better to not compute args if there is a signature and it's correct. The current implementation does it always.
It's not critical for performance but I think it's possible to write it another way without losing readability.

Reply:
We need to compute `args` in order to check `Signature`. Please, look at `checkArgsSignature`. It checks if arguments and their order from `Signature` correspond to original arguments.